src/main/java/hudson/plugins/xvnc/Xvnc.java

```diff
@@ -92,7 +95,9 @@ public void setUp(Context context, Run<?, ?> build, FilePath workspace, Launcher
 
         String cmd = Util.nullify(DESCRIPTOR.xvnc);
         if (cmd == null) {
-            cmd = "vncserver :$DISPLAY_NUMBER -localhost -nolisten tcp";
+            cmd = "vncserver :$DISPLAY_NUMBER -localhost -nolisten tcp " + (this.additionalArgs != null ? this.additionalArgs : "");
+        } else {
+            DESCRIPTOR.xvnc += " " + (this.additionalArgs != null ? this.additionalArgs : "");
```

Comment:
`DESCRIPTOR.xvnc` is a global configuration that should be respected by jobs unless overridden but certainly not modified by every build executed.

It should either be `cmd = DESCRIPTOR.xvnc + ' ' + (this.additionalArgs != null ? this.additionalArgs : "")` or there should be a new macro introduced so the command can be configured to place arguments from job to specific part of command line. I am not sure whether vnc server implementations are sensitive to argument options so I guess we have to presume they are...